packages/core/src/components/forms/_input-group.scss

```diff
@@ -27,6 +27,7 @@ Markup:
 .pt-round - Rounded caps. Button will also be rounded.
 .pt-large - Large group. Children will adjust size accordingly.
 .pt-intent-primary - Primary intent. (All 4 intents are supported.)
+.pt-fill - Take up full width of parent element
```

Comment:
nit: trailing period please (like the lines above)

Reply:
Copied from `_inputs.scss` like this. Will fix.